ui/src/test/java/com/hyperwallet/android/ui/view/widget/DateUtilTest.java

```diff
@@ -0,0 +1,121 @@
+package com.hyperwallet.android.ui.view.widget;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Collection;
+import java.util.List;
+
+@RunWith(RobolectricTestRunner.class)
+public class DateUtilTest {
```

Comment:
test names are not following the naming convention, testMethodName_UseCase
also it would be better to use the ExpectedException if possible

Reply:
@fmattos-hw Thank you. Done